src/site/content/en/metrics/cls/index.md

```diff
@@ -242,6 +242,11 @@ Layout shifts that occur within 500 milliseconds of user input will have the
 [`hadRecentInput`](https://wicg.github.io/layout-instability/#dom-layoutshift-hadrecentinput)
 flag set, so they can be excluded from calculations.
 
+{% Aside 'caution' %}
+  Please note: scrolling input is not given a 500 ms buffer, as per the 
+  [Layout Instability Spec](https://github.com/WICG/layout-instability#recent-input-exclusion)
```

Comment:
I'd make this more generic, how about:

"The `hadRecentInput` flag will only be true for discrete input events like tap, click, or keypress. Continuous interactions such as scrolls, drags, or pinch and zoom gestures are not considered "recent input". See the [Layout Instability spec](https://github.com/WICG/layout-instability#recent-input-exclusion) for more details."

Reply:
+1, changed